db/migrate/20240808175526_migrate_tag_added_from_filter_to_automation.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+class MigrateTagAddedFromFilterToAutomation < ActiveRecord::Migration[7.1]
+  def up
+    if defined?(DiscourseAutomation) &&
```

Comment:
I wonder if this entire logic should be wrapped in a `begin/rescue` b/c we don't really want this migration to FAIL if in the future, some of these classes change or something... It's risky to write migrations with logic referencing specific classes like this. Yeah I think rescueing errors would be good.